### PR TITLE
Dev alex

### PR DESCRIPTION
**Pull request by Khoroshevskyi**
The problem is with NCBI Trace database is fixed.

TODO:
- [x] update changelog

### CODE REVIEW

**Review by nleroy917 — APPROVED** [review]
Nothing crazy. Just those two (or maybe it was three) comments.

File: geofetch/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.9.0"
+__version__ = "0.9.1"
```
Comment (nsheff) [inline geofetch/_version.py:1]: This is enough of a change to warrant `0.10.0`.

**Review by nsheff — APPROVED** [review]
It seems like it didn't turn out to be as big of an issue as you thought -- what was the trick?

File: geofetch/geofetch.py [hunk 2]
```diff
@@ -90,22 +96,25 @@ def __init__(
         sra_folder="",
         bam_conversion=False,
         picard_path="",
-        silent=False,
-        verbosity=None,
-        logdev=False,
+        # silent=False,
```
[the remark this answers is not in the document]
Comment (Khoroshevskyi) [reply geofetch/geofetch.py:99]: Yes, I ll fix it

**Comment by Khoroshevskyi** [comment]
> It seems like it didn't turn out to be as big of an issue as you thought -- what was the trick?

Before we could download all search metadata (all result information) by providing SRP number. Now it's impossible and we have to search and fetch separately. And also we should use xml to do it, as json output isn't json:(. 
The biggest challenge for me was lack of the knowledge about variables that were used to create metadata and what changes could cause this variables in further steps.

**Pull request merged**